yarn.lock

```diff
@@ -1969,17 +1969,17 @@ argparse@^2.0.1:
 
 arr-diff@^4.0.0:
   version "4.0.0"
-  resolved "https://registry.npmjs.org/arr-diff/-/arr-diff-4.0.0.tgz"
+  resolved "https://registry.yarnpkg.com/arr-diff/-/arr-diff-4.0.0.tgz#d6461074febfec71e7e15235761a329a5dc7c520"
```

Comment:
What was the reason for this switch? From `npmjs.org` to `yarnpkg.org` ?

Reply:
This was autogenerated when running `yarn upgrade "package"`. I didn't change it explicitly.

Reply:
Not saying this is wrong, but we are changing something all over the place and this is odd.

There is this interesting SO thread here: https://stackoverflow.com/questions/69526949/yarn-lock-file-resource-source-changed-from-registry-yarnpkg-com-to-registry-npm

(even though in their case it went the other way around)

Looks like a single import/install done differently can change all entries in the lock file.

What are we using in other repos? Is it mostly npmjs.org, or yarnpkg.org?

Reply:
Most repos use `yarnpkg.org`. @alaisgomes might be the best person to tell us if there is any risk with the swirtch.